docs/source/using/troubleshooting.adoc

```diff
@@ -194,3 +194,29 @@ flag in the `minishift start` command. Alternatively, the problem might be
 an outdated version of VirtualBox.
 
 To avoid this issue, it is recommended to use VirtualBox 5.1.12 or later.
+
+[[insufficient-privileges]]
+== Hyper-V: Administrator privileges insufficient to run Hyper-V
+
+If you run Minishift with Hyper-V on Windows, it is recommended that you add yourself to the Hyper-V Administrators group. You do not need to run as an Administrator in your shell.
+
+For example, if use PowerShell you can add yourself to the Hyper-V Administrators group as follows:
+
+. As an administrator run the following command:
++
+----
+([adsi]”WinNT://./Hyper-V Administrators,group”).Add(“WinNT://$env:UserDomain/$env:Username,user”)
+----
+
+. Log off and log in for the change to take effect.
+
+You can also use the GUI to add yourself to the Hyper-V Administrators group as follows:
+
+. Click the *Start* button and choose *Computer Management*.
+. In the *Computer Management* window, select *Local Users And Groups* and then double click on *Groups*.
+. Double click on the *Hyper-V Administrators* group, the *Hyper-V Administrators Properties* dialog box is displayed.
+. Add your account to the Hyper-V Administrators group and log off and log in for the change to take effect.
+
+Now you can run the Hyper-V commands from a console or a shell as a normal user.
```

Comment:
"from a console or a shell as a normal user" - so what's the difference between console and shell? Effectively that's the same thing, right?